fe/fe-core/src/main/java/com/starrocks/planner/ExchangeNode.java

```diff
@@ -202,11 +202,31 @@ protected String debugString() {
 
     @Override
     protected String getNodeExplainString(String detailPrefix, TExplainLevel detailLevel) {
+        StringBuilder output = new StringBuilder();
+        List<Expr> partitionExprs = dataPartition.getPartitionExprs();
+        if (detailLevel == TExplainLevel.VERBOSE) {
+            if (distributionType != null) {
+                output.append(detailPrefix).append("distribution type: ")
+                        .append(distributionType).append('\n');
+            }
+            if (partitionType != null) {
+                output.append(detailPrefix).append("partition type: ")
+                        .append(partitionType).append('\n');
+            }
+            if (partitionExprs != null && !partitionExprs.isEmpty()) {
```

Comment:
can replace it with `CollectionUtils.isNotEmpty(partitionExprs)`

Reply:
updated